weed/storage/backend/rclone_backend/rclone_backend.go

```diff
@@ -0,0 +1,270 @@
+package rclone_backend
+
+import (
+	"context"
+	"fmt"
+	"github.com/rclone/rclone/fs/config/configfile"
+	"github.com/seaweedfs/seaweedfs/weed/util"
+	"io"
+	"os"
+	"time"
+
+	"github.com/google/uuid"
+
+	_ "github.com/rclone/rclone/backend/all"
+	"github.com/rclone/rclone/fs"
+	"github.com/rclone/rclone/fs/accounting"
+	"github.com/rclone/rclone/fs/object"
+
+	"github.com/seaweedfs/seaweedfs/weed/glog"
+	"github.com/seaweedfs/seaweedfs/weed/pb/volume_server_pb"
+	"github.com/seaweedfs/seaweedfs/weed/storage/backend"
+)
+
+func init() {
+	backend.BackendStorageFactories["rclone"] = &RcloneBackendFactory{}
+}
+
+type RcloneBackendFactory struct {
+}
+
+func (factory *RcloneBackendFactory) StorageType() backend.StorageType {
+	return "rclone"
+}
+
+func (factory *RcloneBackendFactory) BuildStorage(configuration backend.StringProperties, configPrefix string, id string) (backend.BackendStorage, error) {
+	return newRcloneBackendStorage(configuration, configPrefix, id)
+}
+
+type RcloneBackendStorage struct {
+	id         string
+	remoteName string
+	fs         fs.Fs
+}
+
+func newRcloneBackendStorage(configuration backend.StringProperties, configPrefix string, id string) (s *RcloneBackendStorage, err error) {
+	s = &RcloneBackendStorage{}
+	s.id = id
+	s.remoteName = configuration.GetString(configPrefix + "remote_name")
+
+	configfile.Install()
```

Comment:
Should this line only be called once?

Reply:
Good suggestion, this also crossed my mind.
I moved it into the `init()` method.